shared/native/notification-listeners.desktop.js

```diff
@@ -18,12 +18,18 @@ var sentNotifications = {}
 let lastLoggedInNotifyUsername = null
 
 // TODO(mm) Move these to their own actions
-export default function (dispatch: Dispatch, notify: any): incomingCallMapType {
+export default function (dispatch: Dispatch, getState: () => Object, notify: any): incomingCallMapType {
   return {
     'keybase.1.NotifySession.loggedOut': params => {
       lastLoggedInNotifyUsername = null
-      notify('Logged out of Keybase')
-      dispatch(logoutDone())
+
+      // Do we actually think we're logged in?
+      if (getState().config &&
```

Comment:
When we error out of provisioning it gives us the error, then we're notified we're logged out. this blows away the error screen so let's just not do this if we think we're already logged out